lib/internal/errors.js

```diff
@@ -147,6 +162,27 @@ function lazyBuffer() {
   return buffer;
 }
 
+const addCodeToName = hideStackFrames(function addCodeToName(err, name, code) {
+  // Set the stack
+  err = captureLargerStackTrace(err);
```

Comment:
This has caused some performance issues for all errors due to overcollecting stack frames where no frames were meant to be hidden.